packages/fxa-settings/src/components/Tooltip/index.tsx

```diff
@@ -51,7 +51,7 @@ export const Tooltip = ({
       className={classNames(
         `z-50 absolute py-2 px-6 text-center text-white
          rounded text-xs font-header font-bold
-          shadow-tooltip-grey-drop
```

Comment:
CSS shapes do not work in HCM mode (see https://accessabilly.com/accessibility-issues-concerning-windows-high-contrast-mode/). For the tooltip, this means the caret is rendered as a filled rectangle. We could consider replacing the caret with an svg shape instead, but I'm not sure the current state is a problem for readability in HCM mode now that a border is added to the tooltip area.